QRCodeBasedMetroTicketingSystem/QRCodeBasedMetroTicketingSystem.Web/wwwroot/js/Scanner.js

```
@@ -0,0 +1,157 @@
+                statusTitle.textContent = 'Invalid Ticket';
+                statusMessage.textContent = result.Message;
+
+                // Add cross icon
+                statusIcon.className = 'fas fa-times-circle text-danger';
+                ticketInfo.style.display = 'none';
+            }
+        } catch (err) {
+            // Exception handling
+            statusCard.className = 'status-card status-error';
+            statusTitle.textContent = 'System Error';
+            statusMessage.textContent = 'Error processing QR code: ' + err.message;
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)